src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -540,7 +539,8 @@ public ChassisSpeeds getRobotRelativeSpeeds() {
   /** Returns the current odometry pose. */
   @AutoLogOutput(key = "Odometry/Robot")
   public Pose2d getPose() {
-    return estimator.getEstimatedPosition();
+    // return estimator.getEstimatedPosition(); //Vision
+    return pose;
```

Comment:
do we want this change?

Reply:
you would probably know that better than me